Sources/NIOPosix/SelectableEventLoop.swift

```diff
@@ -547,6 +579,8 @@ Further information:
                 // Drop everything (but keep the capacity) so we can fill it again on the next iteration.
                 self.tasksCopy.removeAll(keepingCapacity: true)
             }
+            let tickInfo = NIOEventLoopTickInfo(eventLoopID: ObjectIdentifier(self), numberOfTasks: tasksProcessedInTick, startTime: tickStartTime)
```

Comment:
We can optimize this and create the loop ID only once. The optimizer might do this for us, but it's nicer not to rely on it.